dist/backdraft.js

```diff
@@ -575,8 +581,101 @@ _.extend(Plugin.factory, {
         return columnType;
       }
     });
+  },
+
+  _applySavedState: function() {
+    var reportSettings = this.table.options.reportSettings;
+    var column;
+    if (reportSettings) {
+      _.each(reportSettings.columnAdds, function(columnName) {
+        column = this._findColumnByAttr(columnName);
+        if (column) {
+          column.visible = true;
+        }
+      }, this);
+
+      _.each(reportSettings.columnSubtracts, function(columnName) {
+        column = this._findColumnByAttr(columnName);
+        if (column) {
+          column.visible = false;
+        }
+      }, this);
```

Comment:
This section seems to deal exclusively with column visibility. Can we break that out into a separate method? Then the contents of _applySavedState can become: 
```
_applySavedState: function() {
   ... snip... 
   this._setColumnVisibility(reportSettings);
   this._reorderColumns(reportSettings.columnOrder);
}
```